test/unit/history/remove.test.js

```diff
@@ -129,8 +123,6 @@ describe('History::remove', function() {
       let two = history.append(function two() {}, 'resolve')
       history.append(function three() {}, 'resolve')
 
-      history.archive()
-
```

Comment:
not sure what these were doing, specs pass without them so hopefully this is okay...